GameServerLib/Logic/GameObjects/AttackableUnits/AI/ObjAIBase.cs

```diff
@@ -12,22 +17,71 @@ public class ObjAIBase : AttackableUnit
         private object BuffsLock { get; }
         private Dictionary<string, Buff> Buffs { get; }
         
+        private List<UnitCrowdControl> crowdControlList = new List<UnitCrowdControl>();
```

Comment:
`_crowdControlList`

Reply:
It was ``crowdControlList`` already in ``AttackableUnit``.
Considering i would like to keep changes to only moving stuff from AttackablrUnit, i don't think this is a good idea.
Not to mention the whole UnitCrowdControl thing mimics ActionState flags + invulnerability and targetability.
It should be addressed in another PR.